src/importexport/ove/internal/ove.cpp

```diff
@@ -4392,6 +4411,7 @@ bool OvscParse::parse()
     Block placeHolder;
 
     m_handle = &handle;
+    auto _ = resetToNull(&m_handle);
```

Comment:
It seems that this problem should be solved in another way. Better to remove m_handle and add the handle to arguments of BasicParse::readBuffer and BasicParse::jump:

BasicParse::readBuffer(StreamHandle& handle, Block& placeHolder, int size)
BasicParse::jump(StreamHandle& handle, int offset)


This is more explicit and doesn't cause the invalid pointer problem

Reply:
doesn't seem to be an easy or small change